i6_models/parts/conformer/convolution.py

```diff
@@ -85,3 +91,62 @@ def forward(self, tensor: torch.Tensor) -> torch.Tensor:
         tensor = self.pointwise_conv2(tensor)
 
         return self.dropout(tensor)
+
+
+@dataclass
+class ConformerConvolutionV2Config(ConformerConvolutionV1Config):
+    """
+    New attribute:
+        dropout_broadcast_axes: string of axes to which dropout is broadcast, e.g. "T" for broadcasting to the time axis
+                                setting to None to disable broadcasting
+    Allows even kernel size
+    """
+
+    dropout_broadcast_axes: Optional[Literal["B", "T", "BT"]] = None
+
+    def check_valid(self):
+        assert self.kernel_size % 2 == 1, "ConformerConvolutionV1 only supports odd kernel sizes"
+
```

Comment:
Above you say it supports even kernel sizes?
```suggestion
```

Reply:
sorry, I thought it would support but not really and I forgot to revert the docstring.

Reply:
Support for even kernel sizes can be added by manually padding the input tensors correctly. We have a helper function added in https://github.com/rwth-i6/i6_models/pull/54.

Reply:
[torch.nn.Conv1d](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html) supports only one single value for padding both ends. but as you said, we could pad the input by ourselves. Just support for even kernel size is not critical.